src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java

```diff
@@ -121,7 +121,7 @@ public AIState decideWhatToDo()
 
         final int numOfBreedableAnimals = ((int) animals.stream().filter(animal -> animal.getGrowingAge() == 0).count());
 
-        final boolean hasBreedingItem = worker.hasItemInInventory(getBreedingItem(), 0);
+        final boolean hasBreedingItem = worker.hasItemInInventory(getBreedingItem().getItem(), 0);
```

Comment:
Please this. Since this is not ItemStack agnostick.
User InventoryUtils to check for a ItemStack.